listenbrainz/db/dump_manager.py

```diff
@@ -320,6 +322,23 @@ def create_test_parquet_files():
 
         sys.exit(-2)
 
+
+@cli.command(name="import_yim_playlists")
+@click.argument('patch-slug', type=str)
+@click.argument('dump-file', type=str)
+def create_full(patch_slug, dump_file):
```

Comment:
Rename the method.

Reply:
Probably also add a comment that first copy the file to container.

Reply:
Makes sense. Can you please take care of this while you're at it?

Reply:
Already done :)